### ISSUE TITLE
About the accuracy of tgif-qa

### ISSUE THREAD

**Issue by Wanan-ni** [thaolmk54/hcrn-videoqa#7]
Hi，
    I downloaded code、features、pre-trained models, but I got the accuracy of Count about 4.05/4.04/4.05 on test. When I train the model， I got 4.0639/4.0802/4.0599 on Count test and 0.7476/0.7454/0.7449 on Action test. I wonder if the parameters of configs/tgif_qa_xx.yml need to be adjusted, or I need do other settings.

**Comment by thaolmk54** [comment]
Hi,

I'm sorry for the issue you got. 
I'm not really sure what is the problem here. I have asked someone to independently reproduce results from the pretrained models and pretty much it is similar to what in the paper.

As for the configuration files, I do think you need any adjustments. The issue you got possibly comes from the feature joining. Can you please remove the current features in your local storage then re-download and join them all together again to see if it helps? Also, please note that if you train the network yourself the results might vary a bit as PyTorch is not deterministic on different GPUs.

**Comment by Wanan-ni** [comment]
OK，I‘ll try again. Thanks a lot : )
